sympy/printing/pretty/pretty_symbology.py

```diff
@@ -614,6 +614,20 @@ def annotated(letter):
     else:
         return ascii_pics[letter]
 
+def is_combining(sym):
+    """Check whether symbol is a unicode modifier.
+
+    See stringPict.width on usage.
+    """
+    if not pretty_use_unicode():
+        return False
+
+    return True if (u'\N{COMBINING GRAVE ACCENT}' <= sym <=
+                    u'\N{COMBINING LATIN SMALL LETTER X}' or
+
+                    u'\N{COMBINING LEFT HARPOON ABOVE}' <= sym <=
+                    u'\N{COMBINING ASTERISK ABOVE}') else False
+
```

Comment:
Can't you make a test that passes in a symbol that meets these criteria? I'm not sure what sym is? Is it a unicode object?

Reply:
I will look into it. Though it may take some time.

Reply:
I have added the test. Please take a look.